recipes-core/ovmf/ovmf/0003-BaseTools-makefile-adjust-to-build-in-under-bitbake.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/BaseTools/Source/C/Makefiles/header.makefile b/BaseTools/Source/C/Makefiles/header.makefile
```

Comment:
I couldn't get the patch from upstream openembedded to apply because there are some weird non-ascii characters littered in the source so I manually made the changes and did a git diff. Without this patch OVMF doesn't build under openembedded.